packages/cozy-harvest-lib/src/helpers/accounts.js

```diff
@@ -28,3 +40,9 @@ export const mergeAuth = (account, authData) => ({
   ...account,
   auth: merge(account.auth, authData)
 })
+
+export default {
+  getLabel,
+  prepareAccountData,
```

Comment:
s/prepareAccountData/prepareData/ . It is still unclear what prepareData does. What does it do ?